docs/Change_Log.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+#### April 3, 2023
+
+## Interaction Channel Data
+
+Interactions now contain a `channel` field which is a partial channel object and guaranteed to contain `id` and `type`. We recommend that you begin using this channel field to identify the source channel of the interaction, and may deprecate the existing `channel_id` field in the future. See the [interaction documentation](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object) for more details.
```

Comment:
> may deprecate the existing `channel_id` field in the future.

This doesn't agree with the below change which marks `channel_id` as already deprecated.

Reply:
well it's a note that it will be deprecated,
I am not surprised with that, as channel is just a 
better idea, which just has the id in it

Reply:
I meant that this change below marks the field as deprecated, and the changelog says "[Discord] may deprecate the existing `channel_id` field in the future", but its already deprecated: https://github.com/discord/discord-api-docs/pull/6051/files#diff-b44c2bafd9f1fb4cd3383e25770ef07d972796aa56dedd73ba1a7853df2d62d0L22-R23